modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1111,7 +1111,7 @@ public void testAddOrUpdateObjectEntry() throws Exception {
 		_assertCount(1);
 
 		_assertObjectEntryValues(
-			21, values,
+			22, values,
```

Comment:
Why you changed the `expectedValuesSize` if you don't add a new entry?

Reply:
This was to fix an existing test failure. I believe a field was added to the `_objectDefinition` and the number of values was not updated.
I put these changes in a separate commit.